frontend/src/app/components/ForgotPasswordComponents/ForgotPassword.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+import React, { useState } from 'react';
+
+import './ForgotPassword.scss';
+
+//* * TODO functionality */
```

Comment:
What do you want to implement here? I'm just asking if it's a forgotten comment or it's a valid point.

Reply:
Right now, it's just a static page, where user can insert his data, but that's all. We don't have a functional registration process, so I just left the comment here, but I guess I can remove that. We have issues right now:)